commoneventconsumer/src/main/scala/com/gu/notifications/athena/Athena.scala

```diff
@@ -0,0 +1,85 @@
+package com.gu.notifications.athena
```

Comment:
None of this code is new; it's all been moved from https://github.com/guardian/mobile-n10n/blob/8cb80b95f3e191f429815e3755413633c98a13a6/eventconsumer/src/main/scala/com/football/notifications/events/AthenaMetrics.scala.